src/context/useNotes.jsx

```diff
@@ -0,0 +1,45 @@
+import { createContext,useContext,useReducer } from "react";
+
+
+const NotesContext=createContext(null)
+
+export const useNotes=()=>useContext(NotesContext)
+
+const notesReducer=(state,action)=>{
```

Comment:
you should move your reducers into a diff folder

Reply:
Got it.